### PR TITLE
solvedTask

### PR DESCRIPTION
**Pull request by AVykhovanok**

### CODE REVIEW

File: src/main/java/mate/academy/spring/dto/CinemaHallRequestDto.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package mate.academy.spring.dto;
+
+public class CinemaHallRequestDto {
+    private int capacity;
```
Comment (olekskov) [inline src/main/java/mate/academy/spring/dto/CinemaHallRequestDto.java:4]: Change 'int' to 'Integer' for better consistency and null-safety.

File: src/main/java/mate/academy/spring/dto/CinemaHallResponseDto.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package mate.academy.spring.dto;
+
+public class CinemaHallResponseDto {
+    private Long id;
+    private int capacity;
```
Comment (olekskov) [inline src/main/java/mate/academy/spring/dto/CinemaHallResponseDto.java:5]: DTO properties should be objects to clearly identify difference between 0 and missing value (null)

**Review by olekskov — CHANGES_REQUESTED** [review]

File: src/main/java/mate/academy/spring/controller/CinemaHallController.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+    @Autowired
+    public CinemaHallController(CinemaHallService cinemaHallService,
+                                CinemaHallMapper cinemaHallMapper) {
+        this.cinemaHallService = cinemaHallService;
+        this.cinemaHallMapper = cinemaHallMapper;
+    }
+
+    @GetMapping
+    public List<CinemaHallResponseDto> getAllCinemaHalls() {
```
Comment (overpathz) [inline src/main/java/mate/academy/spring/controller/CinemaHallController.java:30]: In different controllers you have different names for mapping methods.
Uniform namings across all controllers to have one style.

image

**Review by overpathz — CHANGES_REQUESTED** [review]
LGTM.
Fix some points from checklist

**Review by Ivan95kos — APPROVED** [review]
Great job

**Pull request closed without merging**